app/uk/gov/hmrc/helptosavefrontend/views/register/confirm_email.scala.html

```diff
@@ -23,19 +23,19 @@
 
 @(email: String, form: Form[ConfirmEmail])(implicit htsContext: HtsContext, request: Request[_], messages: Messages)
 
-@uk.gov.hmrc.helptosavefrontend.views.html.main_template(title = "confirm-email", bodyClasses = None) {
+@uk.gov.hmrc.helptosavefrontend.views.html.main_template(title = "Do you want to use the email address we hold for you? - Help to Save - GOV.UK", bodyClasses = None) {
```

Comment:
this message should also go into messages file ?

Reply:
ditto